README.md

```diff
@@ -1,38 +1,67 @@
 # Thali Cordova Plugin
 
 <Work on progress, not near release quality !!>
```

Comment:
This is the original wording, I haven't changed anything here, but yes, probably makes more sense that way.